src/main/java/core/basesyntax/strategy/impl/BalanceHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceHandler implements OperationHandler {
```

Comment:
Don't forget about the naming convention. Should be `BalanceOperationHandler`
the same to all other handlers